platform/io/platform-io-impl/src/test/java/org/codice/ddf/platform/io/FelixCfgPersistenceStrategyTest.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.platform.io;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.codice.ddf.platform.io.internal.PersistenceStrategy;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class FelixCfgPersistenceStrategyTest {
+
+    private PersistenceStrategy strategy = new FelixCfgPersistenceStrategy();
+
+    private SimplePersistenceTestCase testCase;
+
+    @Before
+    public void before() throws Exception {
+        testCase = new SimplePersistenceTestCase(strategy);
+    }
+
+    @Test
+    public void testGetExtension() {
+        assertThat(strategy.getExtension(), is("cfg"));
+    }
+
+    @Test
+    public void testReadWriteInts() throws Exception {
```

Comment:
❗️ Ints are not supported by the Felix cfg format. So the test should actually validate that a string would be returned.